extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -145,6 +180,20 @@ export class AmpSelector extends AMP.BaseElement {
       this.setSelection_(el);
       this.setInputs_();
     }
+
+    // Trigger 'select' event with two data params:
+    // 'option' - option value of the selected or deselected element.
+    // 'options' - comma-delimited option values of all selected elements.
+    const options = [];
+    this.selectedOptions_.forEach(element => {
+      options.push(element.getAttribute('option'));
```

Comment:
Added a return value to `setInputs_`. I think this is better than maintaining an instance var since it requires less state tracking and avoids confusion about when it's updated.